src/utils/network-schemas.ts

```diff
@@ -28,15 +30,19 @@ export interface NetworkSchema {
     writeToShadow: boolean
   ) => boolean;
   deserialize: (world: HubsWorld, eid: EntityID, data: CursorBuffer) => void;
-  serializeForStorage: (eid: EntityID) => StoredComponent;
-  deserializeFromStorage: (eid: EntityID, storedComponent: StoredComponent) => void;
+  serializeForStorage?: (eid: EntityID) => StoredComponent;
+  deserializeFromStorage?: (eid: EntityID, storedComponent: StoredComponent) => void;
 }
 
 export const schemas: Map<Component, NetworkSchema> = new Map();
 schemas.set(NetworkedMediaFrame, NetworkedMediaFrameSchema);
 schemas.set(NetworkedTransform, NetworkedTransformSchema);
 schemas.set(NetworkedVideo, NetworkedVideoSchema);
 schemas.set(NetworkedWaypoint, NetworkedWaypointSchema);
+schemas.set(NetworkedFloatyObject, {
+  componentName: "floaty-object",
```

Comment:
Should this be `networked-floaty-object`? (I guess the `networked-` prefix might be implied... I wasn't as clever with any of the other ones so they all have `networked-` in their names.)

Reply:
No strong opinions here. Would like to get rid of this somehow as its just another thing you may mess up on copy/paste when making a new component (I almost screwed up matching the two instances of `NetworkedFloatyObject` in this block.